docs/content/integrations/airlift/operator-migration/bash-operator-dbt.mdx

```diff
@@ -0,0 +1,53 @@
+# Operator migration guides: Migrating usage of `BashOperator` for `dbt`
+
+In this page, we'll explain migrating an Airflow `BashOperator` that runs a `dbt` command to Dagster.
+
+### Background
+
+In Airflow, you might have a `BashOperator` that runs a `dbt` command. For example, you might have a task that runs `dbt run` to build your dbt models.
+
+```python file=/integrations/airlift/operator_migration/bash_operator_dbt.py
+from airflow.operators.bash import BashOperator
+
+run_dbt_model = BashOperator(task_id="build_dbt_models", bash_command="dbt run")
+```
+
+### Dagster equivalent
+
+The Dagster equivalent is to instead use the `dagster-dbt` library to run commands against your dbt project. Here would be the equivalent code in Dagster:
+
+```python file=/integrations/airlift/operator_migration/using_dbt_assets.py
+from dagster_dbt import DbtCliResource, DbtProject, dbt_assets
+
+from dagster import AssetExecutionContext
+
+project = DbtProject(project_dir="path/to/dbt_project")
+
+
+@dbt_assets(manifest=project.manifest_path, project=project)
```

Comment:
nit: you don't need to pass project here as it is only required when `DagsterDbtTranslatorSettings.enable_code_references == True`

That said, we should only pass the project to the asset decorator. It was discussed in the past but never done. I just opened a [ticket](https://linear.app/dagster-labs/issue/AD-722/deprecate-manifest-param-in-dbt-assets-in-favor-of-project) to deprecate the manifest param in favor of project.

Reply:
I'll remove the project for now